tools/osv-scanner/cmd/scanner.go

```diff
@@ -103,6 +103,44 @@ func scanGit(repoDir string) (*osv.Query, error) {
 	return osv.MakeCommitRequest(commit), nil
 }
 
+type DockerPackageVersion struct {
+	Name    string
+	Version string
+}
+
+func scanDebianDocker(query *osv.BatchedQuery, dockerImageName string) {
+	cmd := exec.Command("docker", "run", "--rm", dockerImageName, "/usr/bin/dpkg-query", "-f", "${Package}###${Version}\\n", "-W")
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		log.Fatalf("Failed to get stdout: %s", err)
+	}
+	err = cmd.Start()
+	if err != nil {
+		log.Fatalf("Failed to start docker image: %s", err)
+	}
+	defer cmd.Wait()
+	if err != nil {
+		log.Fatalf("Failed to run docker: %s", err)
+	}
+	scanner := bufio.NewScanner(stdout)
+	for scanner.Scan() {
+		text := scanner.Text()
+		text = strings.TrimSpace(text)
+		if len(text) == 0 {
+			continue
+		}
+		splitText := strings.Split(text, "###")
+		pkgDetails := osv.MakePkgDetailsRequest(lockfile.PackageDetails{
+			Name:    splitText[0],
+			Version: splitText[1],
```

Comment:
Can we check for the length of `splitText` here and report the error accordingly?